performance-tools/performance-cluster/Dockerfile

```diff
@@ -64,6 +64,19 @@ RUN echo "deb https://dl.bintray.com/loadimpact/deb stable main" | tee -a /etc/a
 RUN apt-get update
 RUN apt-get install k6
 
+# Install Helm
+ENV HELM_VERSION=v2.0.0
+
+ENV HELM_FILE_NAME helm-${HELM_VERSION}-linux-amd64.tar.gz
+ENV HELM_URL https://storage.googleapis.com/kubernetes-helm/${HELM_FILE_NAME}
+
+RUN curl -o /tmp/$HELM_FILE_NAME ${HELM_URL} \
+  && tar -zxvf /tmp/${HELM_FILE_NAME} -C /tmp \
+  && mv /tmp/linux-amd64/helm /usr/local/bin/helm \
+  && rm -rf /tmp
```

Comment:
Isn't deleting whole `/tmp` directory maybe a little bit too much? Why not only delete the file?